client/src/Admin/Hospitals.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext, useState, useEffect } from 'react';
+import Context from '../Context';
+import ApiService from '../ApiService';
+
+function Hospitals() {
+  const { organization } = useContext(Context);
+  const [hospitals, setHospitals] = useState([]);
+
+  useEffect(() => {
+    ApiService.organizations.get(organization.id).then((response) => {
+      if (organization?.type === 'HEALTHCARE') {
+        setHospitals(response.data.hospitals);
+      }
+    });
+  }, [organization]);
+
+  return (
+    <>
+      {organization.type === 'HEALTHCARE' && (
```

Comment:
The listing of hospitals is a very rudimentary list at the moment. Not much styling is done right now. More can be done if needed in a separate PR